DG Stats/Taunts/Weapon Removal

braids opened the pull request:
I took out any code relevant to the DG stats, DG taunts, and weapon multipliers. For now, this should put the DG plugin in a fairly basic but operational (read: not crashing) state. We'll still have single drinks for DG kills and the synergy bonus, plus whatever special conditions that don't rely on what weapon is involved.

I ran this version on Nucleus and was able to get through a full 25 minutes of capping and drinks with no issues. By the way, dg_add_bots is hella handy.

braids commented:
Much less code removal in this commit, I've left taunts.sp and database.sp untouched. Their methods aren't getting called so they're no threat. Also addressed the merge conflict.

jesseryoung commented on src/drinks.sp:
```diff
@@ -242,7 +245,7 @@ stock DG_Drinks_GivePlayerDeathDrinks(Handle:event, const String:name[]) {
 			new attackerWeapon = TF2_GetCurrentWeapon(attacker);
 			new victimWeapon = TF2_GetCurrentWeapon(victim);
-			if (attackerWeapon != -1 && victimWeapon != -1) {
+			if (attackerWeapon && victimWeapon) {
```
I don't think you wanted to pull this change in?

braids replied:
Whoops, you're right! Looks like that's a commit I missed. Updated this line to Luke's latest change.

braids commented:
I've updated my branch with the requested changes, but when testing out the updated plugin I was still able to get the server crash when a control point was capped. I'm thinking that the DG stats wasn't our root issue. If not that though, then what could it be?

I'll leave this PR as-is for now, I'll be doing more testing later to try and narrow down the cause. So far, the only time it's occurred was when at least two players were on the server, the rest of the server was populated with DG bots, and it occurred on the first capture of the second KOTH map played.

Steps for how I duped it the one time last night:
1) Started server, changed map to koth_harvest_final.
2) Had DefendThat join the server with a DG tag, tested out a few dg chat commands.
3) Capped point back and forth a few times, let the timer run out after 6-7 caps.
4) Next round loaded, added max DG bots. The bots wouldn't move, this is probably due to Harvest missing pathing crap.
5) Changed tf_bot_quota to 0, then changed map to koth_nucleus.
6) Both me and DefendThat load in to Nucleus, I fill server with DG bots again.
7) We run around the map shooting the bots without capping, but eventually one bot get the initial point cap. Server crashes.

I tried repro'ing this later that night, but didn't have another human player on the server. Crash never came, although the point was capped multiple times on Harvest and Nucleus by me and the bots.
